packages/manager/modules/pci/src/projects/project/project.routing.js

```diff
@@ -84,6 +85,17 @@ export default /* @ngInject */ ($stateProvider) => {
         return project.planCode === DISCOVERY_PROJECT_PLANCODE;
       },
 
+      discoveryPromotionVoucherAmount: /* @ngInject */ (
+        pciProjectNew,
+        project,
+      ) =>
+        project.planCode === DISCOVERY_PROJECT_PLANCODE
+          ? pciProjectNew
+              .checkEligibility(DISCOVERY_PROMOTION_VOUCHER)
+              .then((response) => response.voucher?.credit?.text)
+              .catch(() => false)
+          : null,
```

Comment:
Looks like `discoveryPromotionVoucherAmount` is of type string
```suggestion
              .catch(() => '')
          : '',
```